File: kafka/src/main/java/com/packs/kafka/model/JobDetails.java
package com.packs.kafka.model;

import com.datastax.driver.core.utils.UUIDs;
import org.springframework.data.cassandra.core.mapping.PrimaryKey;
import org.springframework.data.cassandra.core.mapping.Table;

import java.sql.Timestamp;
import java.util.UUID;


@Table("job_post_details")
public class JobDetails {

    @PrimaryKey
    private UUID id;

    private Timestamp timestamp;

    private int jobsCount;

    public JobDetails(){}

    public JobDetails(UUID id, Timestamp timestamp, int jobsCount) {
        this.id = id;
        this.timestamp = timestamp;
        this.jobsCount = jobsCount;
    }

    public UUID getId() {
        return id;
    }

    public void setId(UUID id) {
        this.id = id;
    }

    public Timestamp getTimestamp() {
        return timestamp;
    }

    public void setTimestamp(Timestamp timestamp) {
        this.timestamp = timestamp;
    }

    public int getJobsCount() {
        return jobsCount;
    }

    public void setJobsCount(int jobsCount) {
        this.jobsCount = jobsCount;
    }
}

File: ScalaFindWorks/src/main/java/com/packs/ScalaFindWorks/model/JobDetails.scala
package com.packs.ScalaFindWorks.model

import java.sql.Timestamp
import java.util.UUID
import org.springframework.data.cassandra.core.mapping.{PrimaryKey, Table}
import scala.beans.BeanProperty

@Table("job_post_details")
class JobDetails {
  @PrimaryKey
  @BeanProperty
  var id: UUID =

  @BeanProperty
  var timestamp: Timestamp

  @BeanProperty
  var jobsCount: Int =

  def this(id: UUID, timestamp: Timestamp, jobsCount: Int) = {
    this()
    this.id = id
    this.timestamp = timestamp
    this.jobsCount = jobsCount
  }

}
